docs/components/DrawerDocs.js

```diff
@@ -42,6 +42,7 @@ class DrawerDocs extends React.Component {
 
     return (
       <Drawer
+        aria-describedby='description'
```

Comment:
Do you want to also update the `Drawer` instance on line 225 to match these changes?

Reply:
yes, yes I do.  I forgot about that. 👍